src/ErrorHandler.php

```diff
@@ -337,4 +295,32 @@ private function cleanBacktraceFromErrorHandlerFrames(array $backtrace, string $
 
         return $cleanedBacktrace;
     }
+
+    /**
+     * @param \ErrorException $errorAsException The error to be passed onto listeners
+     */
+    private function invokeErrorListeners(\ErrorException $errorAsException): void
+    {
+        foreach ($this->errorListeners as $listener) {
+            try {
+                $listener($errorAsException);
+            } catch (\Throwable $exception) {
+                // Do nothing as this should be as transparent as possible
+            }
+        }
+    }
+
+    /**
+     * @param \Throwable $throwable The exception to be passed onto listeners
+     */
+    private function invokeExceptionListeners(\Throwable $throwable): void
+    {
+        foreach ($this->exceptionListeners as $listener) {
+            try {
+                $listener($throwable);
```

Comment:
Shouldn't we use `call_user_func` here?

Reply:
I agree, since we switched to `callable` this is necessary

Reply:
Nope it's not necessary; I've added a test to prove it: https://github.com/getsentry/sentry-php/pull/762/files#diff-c290a9b0e216f21a0de832495713aa14R313